proto/poktroll/supplier/event.proto

```diff
@@ -6,23 +6,50 @@ option (gogoproto.stable_marshaler_all) = true;
 
 import "cosmos_proto/cosmos.proto";
 import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
 
 import "poktroll/shared/supplier.proto";
 
+enum SupplierUnbondingReason {
+  ELECTIVE = 0;
+  BELOW_MIN_STAKE = 1;
+}
+
 // EventSupplierStaked is emitted with the commitment of the supplier stake message.
```

Comment:
```suggestion
// EventSupplierStaked is emitted when a supplier commits an onchain staking message.
```

Reply:
Happy to rephrase this; however, I think it's important to be clear that the "commit" referenced here is the **specific** point in time when the message is included in a committed block. This isn't really something that the supplier "does", but rather is a consequence thereof. My concern is that phrasing it the way you've suggested either gives that impression or at least makes the timing ambiguous (broadcast vs. commit), reducing clarity.

Wdyt about:
```suggestion
// EventSupplierStaked is emitted when a supplier stake message is committed on-chain.
```

Reply:
LGTM